### DISCUSSION TITLE
ssh后无法输入中文(Cannot input Chinese characters)

### DISCUSSION THREAD

**Discussion by Galaxy-Explosion** [electerm/electerm#2746] [Q&A]
远程服务器的编码为gbk，调整了终端编码为gbk后仍无法输入中文，但是文件中原有的中文能够正常显示。请问这个问题需要怎么配置？
The chrset of the remote server is GBK, and after adjusting my terminal chrset to GBK, you still cannot input the Chinese characters, but the Chinese characters in the file can be displayed correctly. How to solve?

**Comment by zxdong262** [comment]
方便提交issue吗，按issue模版提交可以提供比较全面的信息
